metrics/metrics.go

```diff
@@ -0,0 +1,171 @@
+package metrics
+
+import (
+	"context"
+	"time"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+
+	rpcmetrics "github.com/filecoin-project/go-jsonrpc/metrics"
+
+	"github.com/filecoin-project/lotus/blockstore"
+)
+
+// Global Tags
+var (
+	// common
+	Version, _  = tag.NewKey("version")
+	Commit, _   = tag.NewKey("commit")
+	NodeType, _ = tag.NewKey("node_type")
+	PeerID, _   = tag.NewKey("peer_id")
+
+	// rcmgr
+	ServiceID, _  = tag.NewKey("svc")
+	ProtocolID, _ = tag.NewKey("proto")
+	Direction, _  = tag.NewKey("direction")
+	UseFD, _      = tag.NewKey("use_fd")
+)
+
+// Measures
+var (
+	// common
+	LotusInfo          = stats.Int64("info", "Arbitrary counter to tag lotus info to", stats.UnitDimensionless)
+	PeerCount          = stats.Int64("peer/count", "Current number of FIL peers", stats.UnitDimensionless)
+	APIRequestDuration = stats.Float64("api/request_duration_ms", "Duration of API requests", stats.UnitMilliseconds)
+
+	// rcmgr
+	RcmgrAllowConn      = stats.Int64("rcmgr/allow_conn", "Number of allowed connections", stats.UnitDimensionless)
```

Comment:
@alvin-reyes Can we consolidate these metrics so that the action is a label, i.e. `rcmgr/conns` is the metric and it has the tags of `block` and `allow`. This applies to all the listed metrics below as well.

Reply:
See this issue for more context: https://github.com/application-research/estuary/issues/97

Reply:
Resolved on the latest code change.